javaparser-core/src/main/java/com/github/javaparser/ast/expr/CastExpr.java

```diff
@@ -51,7 +54,7 @@ public CastExpr(Type<?> type, Expression expr) {
     public CastExpr(Range range, Type<?> type, Expression expr) {
```

Comment:
I was planning to keep the abbreviations for names that are not just the word itself: class Expression, class CastExpr.